codeSnippets/snippets/tutorial-kotlin-rpc-client/src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,60 @@
+package com.example
+
+import com.example.model.Pizza
+import com.example.model.PizzaShop
+import io.ktor.client.*
+import io.ktor.client.plugins.websocket.*
+import io.ktor.http.*
+import kotlinx.coroutines.runBlocking
+import kotlinx.rpc.client.withService
+import kotlinx.rpc.serialization.json
+import kotlinx.rpc.transport.ktor.client.rpc
+import kotlinx.rpc.transport.ktor.client.rpcConfig
+import kotlinx.rpc.streamScoped
+
+fun main() = runBlocking {
+    val ktorClient = HttpClient {
+        install(WebSockets)
+    }
```

Comment:
We have `installRPC` function instead, see https://kotlin.github.io/kotlinx-rpc/transport.html#client